### ISSUE TITLE
Getting started and basic examples

### ISSUE THREAD

**Issue by joeandrews** [iamralpht/slalom#8]
Great work on this. Can you provide a quick getting started guide for slalom. I had hacked together some stuff from the examples over on your blog, but I think slalom.js removes some of those dependancies if I am not mistaken.

Thanks

Joe

**Comment by iamralpht** [comment]
Yeah, a getting started guide is a good idea. Until that exists, the
examples in git are the best guidance. Since publishing the article I've
switched to using Hammer.js for gesture recognition (the article code would
just steal all of the touch events, so you couldn't put a button or
anything else in a container managed by slalom) and have added a simple
JSON-based description language. There are also a couple of new examples,
like the pager.
